packages/node_modules/@webex/internal-plugin-wdm/src/device/device.js

```diff
@@ -239,18 +239,6 @@ const Device = WebexPlugin.extend({
     return this.serviceCatalog.resetAllAndRetry(url);
   },
 
-  // this function is exposed beyond the device file
```

Comment:
Removing this might be beyond the scope of the ticket. We have some other wdm tickets to remove these methods.

Reply:
I second this, as it should be handled by [this pull request](https://github.com/webex/webex-js-sdk/pull/1413)